### PR TITLE
Enforce more strict code formatting

### PR DESCRIPTION
**Pull request by ljacobson64**
This enforces more strict code formatting. The first commit shows the changes to the astyle_google options, and the second commit formats all the code in the repo.

### CODE REVIEW

File: .run_tests.sh [hunk 1]
```diff
@@ -44,6 +44,18 @@ cd ..
...
+                    --pad-header                \
+                    --unpad-paren               \
+                    --convert-tabs              \
+                    --align-pointer=type        \
+                    --suffix=none               \
+                    --recursive                 \
+                    --formatted                 \
+                    --lineend=linux             \
+                    --exclude=gtest             \
+                    *.cc *.cpp *.h *.hh *.hpp
```
Comment (makeclean) [inline .run_tests.sh]: Could you add quote marks around the asterisked items, i.e. "*.cc" "*.cpp" "*.h" etc?

Comment (ljacobson64) [reply .run_tests.sh]: How come? if you don't mind me asking

Comment (makeclean) [reply .run_tests.sh]: Without quote marks barfs on a mac

Comment (ljacobson64) [reply .run_tests.sh]: That is a good reason

**Comment by ljacobson64** [comment]
Closing this, will re-open when it's actually ready

**Comment by ljacobson64** [comment]
Specifically, I'm waiting for [this bug](https://sourceforge.net/p/astyle/bugs/423) to get fixed so we can use the --max-code-length option.

**Pull request closed without merging**